app.js

```diff
@@ -5,61 +5,41 @@ const oldBooks = JSON.parse(localStorage.getItem('booksArray')) || [];
 function grab(e) {
   return document.getElementById(e);
 }
-function Book(title, author) {
-  this.key = oldBooks.length;
-  this.title = title;
-  this.author = author;
-}
 
-function deleteBook(index) {
-  window.location.reload();
-  oldBooks.splice(index, 1);
-  localStorage.setItem('booksArray', JSON.stringify(oldBooks));
-  grab('books-container').innerHTML = '';
-  let c = 0;
-  oldBooks.forEach((bookItem) => {
-    oldBooks[c].key = c;
-    template.innerHTML = `
-                <h5>${bookItem.title}</h5>
-                <h5>${bookItem.author}</h5>
-                <button onclick='deleteBook(${c})'>Remove</button>`;
-    const far = template.content.firstElementChild;
-    grab('books-container').appendChild(far);
-    c += c + 1;
-  });
-}
-function addBook() {
-  window.location.reload();
-  const title = form.booktitle;
-  const author = form.bookauthor;
-  const bookItem = new Book(title.value, author.value);
-  oldBooks.push(bookItem);
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-  localStorage.setItem('booksArray', JSON.stringify(oldBooks));
+  deleteBook(index) {
+    oldBooks.splice(index, 1);
+    localStorage.setItem('booksArray', JSON.stringify(oldBooks));
+    grab('books-container').innerHTML = '';
+    initialLoad();
+  }
 
-  template.innerHTML = `<li>
-                <h5>${bookItem.title}</h5>
-                <h5>${bookItem.author}</h5>
-                <button onclick='deleteBook(${bookItem.key})'>Remove</button>
-            </li>`;
-  console.log(oldBooks);
-  const far = template.content.firstElementChild;
-  grab('books-container').appendChild(far);
-  document.forms['books-form'].reset();
-}
+  addBook() {
+    const title = form.booktitle;
+    const author = form.bookauthor;
+    const bookItem = new Book(title.value, author.value);
+    oldBooks.push(bookItem);
```

Comment:
- [ ] [optional] It would be nice to use array methods that don't mutate the array e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)